products/llbuildSwift/CoreBindings.swift

```diff
@@ -237,6 +237,18 @@ public protocol BuildEngineDelegate {
     /// requested Key cannot be supplied, the delegate should provide a dummy rule
     /// that the client can translate into an error.
     func lookupRule(_ key: Key) -> Rule
+
+    /// Handle error message reported by the engine
+    ///
+    /// The delegate should handle this, but a default implementation is
+    /// is provided that does nothing for backward compatiblity.
+    func error(_ message: String)
+}
+
+extension BuildEngineDelegate {
+    func error(_ message: String) {
+        return
```

Comment:
nit: not required

Reply:
Is it not?  I don't want to break SwiftPM downstream by merging this with a newly required method.

Reply:
Sorry, I meant the `return` keyword is not required 😅

Reply:
Ahh, that makes sense.